.gitignore

```diff
@@ -1,4 +1,5 @@
 *.iml
 *.xml
```

Comment:
Please explain why do you ignore all files with extension *.xml?

Reply:
I ignored this files everytime.. need i deleted this line in .gitignore?

Reply:
Do you want to ignore pom.xml?

Reply:
i will delete *.xml